src/components/logo/Logo.jsx

```diff
@@ -32,9 +48,17 @@ export const TYPE = {
 
 export const LOGOS = {
   [TYPE.BRAINLY]: 'brainly-761d75d6ea',
+  [TYPE.BRAINLY_MOBILE]: 'brainly-mobile-d9f7a24c8b',
+  // [TYPE.BRAINLY_MOBILE_NEW]: 'brainly-mobile-new-6879551770',
```

Comment:
for new icons fingerprints are missing, we are using production ones here, so we need to deploy it first.